Runtimes/Core/CMakeLists.txt

```diff
@@ -191,7 +191,6 @@ add_compile_options(
   "$<$<COMPILE_LANGUAGE:Swift>:SHELL:-enable-experimental-feature BitwiseCopyable>"
   "$<$<COMPILE_LANGUAGE:Swift>:SHELL:-enable-experimental-feature Extern>"
   "$<$<COMPILE_LANGUAGE:Swift>:SHELL:-enable-experimental-feature ValueGenerics>"
-  "$<$<COMPILE_LANGUAGE:Swift>:SHELL:-enable-experimental-feature AddressableParameters>"
```

Comment:
I believe this may be a diff against an old commit, since we moved these flags in #79677 -- you may want to consider removing this from `Runtimes/Core/cmake/modules/ExperimentalFeatures.cmake` instead

Reply:
Will do, thanks.